roles/openshift_examples/tasks/modify_imagestreams.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Get remote imagestream json file
+  slurp:
+    src: "{{ openshift_example_remote_imagestream }}"
+  register: imagestreams_encoded
+
+- set_fact:
+    openshift_examples_imagestreams: "{{ (imagestreams_encoded['content'] | b64decode | from_json) }}"
+
+# For each ImageStream item
+# {% for is in openshift_examples_imagestreams['items'] %}
+#
+# For each ImageStream spec tag override the referencePolicy
+# {% for tag in is['spec']['tags'] %}{{ updatedtags.append(tag | combine({'referencePolicy': {'type': 'Source'}})) }}
+#
+# For each ImageStream spec override the tags
+# {% set updatedspec = (is['spec'] | combine({'tags': updatedtags })) %}
+#
+# For each ImageStream override the spec
+# {% set updatedimagestream = (is | combine({'spec': updatedspec})) %}
+#
+# Finally append each modified ImageStream to updated and return the ImageStream list
+# {% set _ = updatedislist.append(updatedimagestream) %}
+# {{ updatedislist }}
```

Comment:
I'm a bit confused--what is the purpose of these comments?

Reply:
@patrickdillon I removed them.  I was thinking it might be easier to understand what was going on for someone in the future but I think it just made a mess.